src/components/pickup/Tabs.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Pickedup from "./Pickedup";
+import Pickup from "./Pickup";
+import { Order } from "../../types/Order";
+
+interface TabPanelProps {
+    children?: React.ReactNode;
+    index: number;
+    value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
```

Comment:
`TabPanel({ { children, value, index }: TabPanelProps) {`

Erase props destructure

Reply:
removed other and shifted props destructuring to the functions formal parameters. This is different than what you state, but I think it might be a good syntax. Let me know if there's still something wrong.